pulsar-broker/src/test/java/org/apache/pulsar/broker/admin/TopicAutoCreationTest.java

```diff
@@ -55,6 +55,7 @@ protected void setup() throws Exception {
         conf.setAllowAutoTopicCreationType(TopicType.PARTITIONED);
         conf.setAllowAutoTopicCreation(true);
         conf.setDefaultNumPartitions(3);
+        conf.setSystemTopicEnabled(false);
```

Comment:
@mattisonchao Do you know why we should disable the system topics? I tried it on my laptop without this line. Get topic policy will return system topic does not exist. Just want to to confirm we will not introduce any break change.

Reply:
>Do you know why we should disable the system topics? 

Because the deleted namespace doesn't allow any topic creation. 

I deleted this line and used another exception judgement to verify it.